libzazen/cuboid_window.c

```diff
@@ -151,12 +199,11 @@ static const char* fragment_shader =
     "}\n";
 
 static void zazen_cuboid_window_update_vertex_buffer(
-    struct zazen_cuboid_window* cuboid_window)
+    struct zazen_cuboid_window* cuboid_window, float l)
```

Comment:
```suggestion
    struct zazen_cuboid_window* cuboid_window, float window_frame_aspect)
 {
   float l = window_frame_aspect
```
とか少し説明的な変数名にしたほうが良いかも
あるいはコメントで説明するとか？